packages/client/src/views/RegisterForm/review/ReviewSection.tsx

```diff
@@ -1804,8 +1804,8 @@ class ReviewSectionComp extends React.Component<FullProps, State> {
                             label={sec.title}
                             action={
                               sec.action &&
-                              declaration.registrationStatus !==
-                                SUBMISSION_STATUS.CORRECTION_REQUESTED && (
+                              (declaration.registrationStatus as string) !==
+                                (SUBMISSION_STATUS.CORRECTION_REQUESTED as string) && (
```

Comment:
```suggestion
                              declaration.registrationStatus !==
                                RegStatus.CorrectionRequested && (
```

Reply:
Thanks @Zangetsu101, I'm not sure how I missed this feedback